src/daq_config_server/log.py

```diff
@@ -1,24 +1,80 @@
 import logging
+from typing import Literal, TextIO
 
+from graypy import GELFTCPHandler
+from pydantic import BaseModel, HttpUrl
 
-# See https://github.com/DiamondLightSource/daq-config-server/issues/73
-# for making the logging configurable
-def get_default_logger(name: str = __name__) -> logging.Logger:
-    logger = logging.getLogger(name)
-    logger.setLevel(logging.INFO)
+LogLevel = Literal["NOTSET", "DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"]
 
-    # Prevent adding handlers multiple times
-    if not logger.handlers:
-        console_handler = logging.StreamHandler()
-        formatter = logging.Formatter(
-            "%(asctime)s - %(levelname)s - %(message)s", datefmt="%Y-%m-%d %H:%M:%S"
-        )
-        console_handler.setFormatter(formatter)
-        logger.addHandler(console_handler)
 
-    return logger
+class GraylogConfig(BaseModel):
+    enabled: bool = False
+    level: LogLevel = "INFO"
+    url: HttpUrl = HttpUrl("http://localhost:5555")
```

Comment:
nit because the protocol is wrong but for completeness, this is also wrong in blueapi, because we're both using the TCPHandler it should be
```suggestion
    url: AnyUrl = HttpUrl("tcp://localhost:5555")
```